src/pkg/services/m365/groups_test.go

```diff
@@ -33,16 +35,15 @@ func TestGroupsIntgSuite(t *testing.T) {
 
 func (suite *GroupsIntgSuite) SetupSuite() {
 	t := suite.T()
+	var err error
```

Comment:
nit: could you please move this declaration closer to the actual usage. I has goldfish memory and forget stuff like this when it's not nearby

Reply:
![goldfish memory](https://thegoldfishtank.com/wp-content/uploads/2013/05/goldfish-memory2.jpg)